server_admin/topics/admin-cli.adoc

```diff
@@ -1,1320 +1,1702 @@
+For example, on:
 
-Or on Windows:
+* Linux:
+[options="nowrap"]
+----
+$ kcadm.sh update clients/c7b8547f-e748-4333-95d0-410b76b3f4a3 -r demorealm -s enabled=false -s publicClient=true -s 'redirectUris=["http://localhost:8080/myapp/*"]' -s baseUrl=http://localhost:8080/myapp -s adminUrl=http://localhost:8080/myapp
+----
+* Windows:
+[options="nowrap"]
+----
+c:\> kcadm update clients/c7b8547f-e748-4333-95d0-410b76b3f4a3 -r demorealm -s enabled=false -s publicClient=true -s "redirectUris=[\"http://localhost:8080/myapp/*\"]" -s baseUrl=http://localhost:8080/myapp -s adminUrl=http://localhost:8080/myapp
+----
 
-    c:\> kcadm update clients/c7b8547f-e748-4333-95d0-410b76b3f4a3 -r demorealm -s enabled=false -s publicClient=true -s "redirectUris=[\"http://localhost:8080/myapp/*\"]" -s baseUrl=http://localhost:8080/myapp -s adminUrl=http://localhost:8080/myapp
 
-Deleting a client::
+==== Deleting a client
 
-Use `delete` operation with the same endpoint uri as for getting a specific client. For example:
+Use the [command]`delete` command with the same endpoint URI that you used to get a specific client.
 
-    $ kcadm.sh delete clients/c7b8547f-e748-4333-95d0-410b76b3f4a3 -r demorealm
+For example:
+[options="nowrap"]
+----
+$ kcadm.sh delete clients/c7b8547f-e748-4333-95d0-410b76b3f4a3 -r demorealm
+----
 
 
 === User operations
 
-Creating a user::
-
-To create a new user perform `create` operation on `users` endpoint. For example:
-
-    $ kcadm.sh create users -r demorealm -s username=testuser -s enabled=true
-
-
-Listing users::
-
-Use `users` endpoint to list users. Number of users may be large, and you may want to limit how many are returned:
-
-    $ kcadm.sh get users -r demorealm --offset 0 --limit 1000
 
-It's also possible to filter users by `username`, `firstName`, `lastName`, or `email`. For example:
+==== Creating a user
 
-    $ kcadm.sh get users -r demorealm -q email=google.com
-    $ kcadm.sh get users -r demorealm -q username=testuser
-
-Note that filtering doesn't use exact matching. For example, the above would match the value of `username` attribute against '\*testuser*' pattern.
-
-You can also filter across multiple attributes by specifying multiple `-q` options, which would return only users
-that match condition for all the attributes.
-
-
-Getting a specific user::
-
-Use user's `id` to compose an endpoint uri - `users/USER_ID`.
+Run the [command]`create` command on the [filename]`users` endpoint to create a new user.
 
 For example:
-
-    $ kcadm.sh get users/0ba7a3fd-6fd8-48cd-a60b-2e8fd82d56e2 -r demorealm
+[options="nowrap"]
+----
+$ kcadm.sh create users -r demorealm -s username=testuser -s enabled=true
+----
 
 
-Updating a user::
+==== Listing users
 
-Use `update` operation with the same endpoint uri as for getting a specific user. For example on Linux:
+Use the [filename]`users
```

Comment:
Typo - repeated word: 'If you want want'

Reply:
Line 1665. Nice catch!

Reply:
typo corrected